Dockerfile.template

```diff
@@ -49,4 +49,7 @@ RUN poetry config settings.virtualenvs.create false \
 # Add binaries' path to PATH.
 ENV PATH="/murakami/bin:${PATH}"
 
-ENTRYPOINT [ "python", "-m", "murakami" ]
+# Mount the expected system path for saving data
+VOLUME ["/data/", "/var/lib/murakami"]
+
+ENTRYPOINT [ "murakami", "-c", "/murakami/murakami.toml" ]
```

Comment:
I think the better way to go would be to use the `-d` option to put the state file in `/data` if that's where it needs to go by default. So this line would read `ENTRYPOINT [ "murakami", "-d", "/data/config.json", "-c", "/murakami/murakami.toml" ]`.

Also, though we may very well want to have this anyway to provide some consistency between the Dockerfiles, the dynamic state file is only necessary when Webthings is enabled, which presumably wouldn't often be the case for the Balena deployment model.

Reply:
Per the previous comment, if using `-d` in the entrypoint works then we should do that and not mount a volume. I also agree that if a device is Balena managed, we should just recommend not turning on the WT support.